Block/Adminhtml/Shopcart/Abandoned/Grid.php

```diff
@@ -37,7 +37,7 @@ public function __construct(
     ) {
         $this->jsonSerializer = $jsonSerializer;
         $this->configHelper = $configHelper;
-        parent::__construct($context, $backendHelper, $quotesFactory, $data);
+        parent::__construct($context, $backendHelper, $quotesFactory, null, null, $data);
```

Comment:
Bro, you need to check the parent. You have to pass $urlDecoder and $parameters, not null. It will not break because parent constructs  them anyways in case of null. Still better to keep the flexibility and do it correctly.